app/glossary.md

```diff
@@ -48,6 +51,9 @@ Cycle
 Database of Qualified Teachers (DQT)
 : A database maintained by the Teacher Regulation Agency. Teachers can sign in to view their teacher record, get copies of QTS and other teaching related qualifications and update their personal and employment details.
 
+DBS Check
+: A service from the Disclosure and Barring Service which checks for any potential criminal record. There are two levels, a ‘basic’ check and an ‘enhanced’ check. Applying for teacher training requires the enhanced check. Generally, the acronym does not need to be spelled out, but the check should be described, for example as a criminal record check.
```

Comment:
I think there's actually 3 levels! https://www.gov.uk/government/organisations/disclosure-and-barring-service/about#dbs-checks

Reply:
Or even 4?!

Reply:
Have updated.